plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -76,12 +77,12 @@ export class PersonState {
         this.updateIsIdentified = false
     }
 
-    async update(): Promise<Person | undefined> {
```

Comment:
The return value wasn't used anywhere and wasn't intuitive, what it was (specifically that we'd return undefined if we didn't need to do any update, but the person exists fine and when there was a problem with updating).

Reply:
It is being used in my subsequent PR-s for moving where person property updating is occurring.

For this reason, delaying reviewing this as well.